### ISSUE TITLE
Clicking scrollbar arrows results in crash

### ISSUE THREAD

**Issue by rnkn** [monocasual/giada#53]
Clicking on any scrollbar arrows will reliably crash Giada.

Giada 0.10.2
OS X 10.11.1

**Comment by rnkn** [comment]
Apologies for not including the crash log previously...
~~~
Process:               giada_osx [44678]
Path:                  /Applications/giada.app/Contents/MacOS/giada_osx
Identifier:            giada_osx
Version:               0.10.2 (1.0)
Code Type:             X86 (Native)
Parent Process:        ??? [1]
Responsible:           giada_osx [44678]
User ID:               501

Date/Time:             2015-11-07 18:38:47.793 +1000
OS Version:            Mac OS X 10.11.1 (15B42)
Report Version:        11
Anonymous UUID:        5CCF7D0C-B2A9-DD78-1FAD-665D4385E042

Sleep/Wake UUID:       989A0753-3EC4-4C14-B1E1-6AE065918E18

Time Awake Since Boot: 360000 seconds
Time Since Wake:       360 seconds

System Integrity Protection: enabled

Crashed Thread:        Unknown

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Backtrace not available

Unknown thread crashed with X86 Thread State (32-bit):
  eax: 0x00000588  ebx: 0x000e5b0b  ecx: 0x00000588  edx: 0x00000039
  edi: 0x00e23f1c  esi: 0x000000ab  ebp: 0xbfffeea8  esp: 0xbfffee7c
   ss: 0x00000023  efl: 0x00010246  eip: 0x00000000   cs: 0x0000001b
   ds: 0x00000023   es: 0x00000023   fs: 0x00000000   gs: 0x0000000f
  cr2: 0x00000000
  
Logical CPU:     4
Error Code:      0x00000014
Trap Number:     14


Binary images description not available


External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 1
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 753110
    thread_create: 0
    thread_set_state: 0

Model: MacBookPro11,2, BootROM MBP112.0138.B16, 4 processors, Intel Core i7, 2.2 GHz, 16 GB, SMC 2.18f15
Graphics: Intel Iris Pro, Intel Iris Pro, Built-In
Memory Module: BANK 0/DIMM0, 8 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343147533641465238412D50422020
Memory Module: BANK 1/DIMM0, 8 GB, DDR3, 1600 MHz, 0x80AD, 0x484D54343147533641465238412D50422020
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x134), Broadcom BCM43xx 1.0 (7.21.94.29.1a4)
Bluetooth: Version 4.4.2f1 16391, 3 services, 27 devices, 1 incoming serial ports
Network Service: iPhone, Ethernet, en4
Serial ATA Device: APPLE SSD SM0256F, 251 GB
USB Device: USB 3.0 Bus
USB Device: Apple Internal Keyboard / Trackpad
USB Device: BRCM20702 Hub
USB Device: Bluetooth USB Host Controller
USB Device: iPhone
Thunderbolt Bus: MacBook Pro, Apple Inc., 17.1
~~~

**Comment by monocasual** [comment]
This bug seems related to custom scrollbars and their draw method based on G_BOX.

**Comment by monocasual** [comment]
How to draw custom scrollbars properly in FLTK: http://www.fltk.org/articles.php?L697

**Comment by monocasual** [comment]
Fixed in 0.13.2 (development).
